addons/account/tests/common.py

```diff
@@ -1013,6 +1023,15 @@ def init_invoice(cls, move_type, partner=None, invoice_date=None):
             line_form.product_id = cls.product_b
         return move_form.save()
 
+
+@tagged('post_install', '-at_install')
+class AccountTestInvoicingCommon(SavepointCase, AccountTestSetup):
```

Comment:
the file already contains a lot of classes. Please docstring (and/or cleanup now or later)

Reply:
most of them will be removed by https://github.com/odoo/odoo/pull/54906